web/docs/auth/migrate-from-11.md

```diff
@@ -0,0 +1,349 @@
+
+As soon as you deploy the new auth system, new users will be able to sign up, but existing users won't be able to log in until you run the migration script on the production database.
+
+You should probably back up your database before migrating a deployed app. Testing the changes locally or on a staging environment first is also a good idea.
+:::
+
+### 1. Migrate to the New Auth System
+
+If you have an existing app with authentication set up, you can follow these steps to migrate to the new auth system:
+
+1. Upgrade Wasp to the latest 0.12 version.
+1. Bump the version to `^0.12.0` in `main.wasp`.
+1. Create the new auth tables in the database by running:
+
+  ```bash
+  wasp db migrate-dev
+  ```
+
+  You should see the new `Auth`, `AuthIdentity` and `Session` tables in your database.
+
+1. Create a data migration script in `src/server/migrateToNewAuth.ts`
+
+    - Below you can find [examples of migration scripts](#example-migration-scripts) for each of the auth methods. They should be fine to use as-is, but you can also modify them to your needs.
+
+1. Add the migration script(s) to the `db.seeds` config:
+  ```wasp title="main.wasp"
+  app myApp {
+    wasp: {
+      version: "^0.12.0"
+    },
+    // ...
+    db: {
+      seeds: [
+        import { migrateEmailAuth } from "@server/migrateToNewAuth.js",
+        import { migrateGoogleAuth } from "@server/migrateToNewAuth.js",
+      ]
+    },
+  }
+  ```
+1. Run the migration script(s) by running:
+  ```bash
+  wasp db seed
+  ```
+  If you added multiple migration scripts, you can pick which one to run by selecting it from the list.
+
+1. Verify that the auth still works locally by logging in with each of the auth methods.
+1. Update your JS code to use the new auth helper functions for getting the `email` or `username` of the currently logged-in user. Read more about the helpers in the [Auth Entities](./entities#accessing-the-auth-fields) section.
+
+  The helpers you'll probably use are the `getEmail` and `getUsername` helpers.
+
+  :::info Migrating a deployed app
+
+    After you have run the migration script locally, updated your JS code and saw that it's okay, _deploy the app_ with the changes you have made so far.
+
+    Then you'll need to run the migration script on the production database with `wasp db seed` command.
+
+    **Running the migration script on the production database**
+
+    We wrote instructions on how to do it for **Fly.io** deployments here: https://github.com/wasp-lang/wasp/issues/1464 
+
+    The instructions should be similar for other deployment providers: setting up some sort of an SSH tunnel from your local machine to the production database and running the migration script locally with `DATABASE_URL` pointing to the production database.
+  :::
+
+
+### 2. Cleanup the Old Auth System
+
+To finish the migration, we need to clean up the old auth system:
```

Comment:
```suggestion
Our app is now working correctly and using new auth, but to completely finish the migration, we need to clean up the old auth system:
```

Reply:
We want to say "your" vs. "our" in our docs when we are referring to their app. I'll implement this suggestion 👍